eZ/Bundle/EzPublishCoreBundle/EventListener/ConsoleCommandListener.php

```diff
@@ -42,9 +48,10 @@ class ConsoleCommandListener implements EventSubscriberInterface, SiteAccessAwar
     /**
      * ConsoleCommandListener constructor.
      */
-    public function __construct($defaultSiteAccessName, EventDispatcherInterface $eventDispatcher)
+    public function __construct($defaultSiteAccess, $siteAccessList, EventDispatcherInterface $eventDispatcher)
```

Comment:
I guess he reflect the change in config name used to fix the issue regarding default site access on commands.